backend/src/controllers/questionGroupAttempt.js

```diff
@@ -15,40 +15,27 @@ const getQuestionGroupAttempts = async (req, res) => {
   } else {
     questionGroupAttempts = await db.QuestionGroupAttempt.findAll({
       where: {
-        '$QuestionGroup.lesson.Course.Class.teachers.id$': teacher.id,
+        '$classes.teachers.id$': teacher.id,
```

Comment:
dit is wss een gevolg van n foute relationship `as`  maar dit zou enkelvoud `class` moeten zijn. Staat wss ook fout in de db.js